product-features/self-service-listing.md

```diff
@@ -1,122 +1,176 @@
 ---
-coverY: 0
-layout:
-  cover:
-    visible: false
-    size: full
-  title:
-    visible: true
-  description:
-    visible: false
-  tableOfContents:
-    visible: true
-  outline:
-    visible: true
-  pagination:
-    visible: true
+description: Create your own pool and make your token tradeable on ALEX decentralized exchange in simple steps!
 ---
 
 # 📝 Self-Service Listing
 
-## Self-Service Listing Procedure
+{% hint style="warning" %}
+**Supported Tokens:** ALEX Self-Service Listing currently supports Stacks Chain Tokens (SIP-010 Standard Token).
+{% endhint %}
 
-Step 1: Submit the token information & deposit the anchor token (Token Y)
+## 🚀 Getting Started
 
-Step 2: Waiting for the pool contract creation (around 24-48 hours)
+### How it works?
 
-Step 3: Deposit the listing token (Token X)
+Self-Service Listing allows you to **create your own liquidity pool** on the ALEX DEX, enabling the **permissionless trade** of the **listed token** with an **anchor token** within the exchange. The anchor token is typically one with a stable value, providing a reliable reference point for defining the price of the newly listed token.
 
-Step 4: Open Pool
+Pool creation usually takes between 24 to 48 hours. Once the pool is created and live, the price discovery phase begins: users can start trading the listed token against the anchor token and viceversa. Users interested in providing liquidity can contribute to the pool like any other ALEX pool.
 
-Step 5: Provide additional token information
+The pool owner is the initial liquidity provider and will receive the corresponding LP tokens upon successful pool creation. Once the pool is live and operational, the owner can withdraw funds just like any other liquidity provider.
 
-***
+{% hint style="info" %}
+Avaiblable Anchor Tokens: Native _STX_ currency, _ALEX_ token and _aBTC_ token.
+{% endhint %}
 
-## Example: Self-service listing for “STX-PEPE” pool
+The trading pool operates under the [ALEX Automated Market Maker (AMM)](../../detailed-information/alexs-automated-market-maker-amm.md) algorithm, which dynamically determines the exchange rate (price) based on the trades.
 
-### Step1：Submit the token information & deposit the anchor token
+### Minimum requirements
 
-#### **Step 1.1 Input the SIP-10 token contract address**
+👉 **Token Deployment.** Have your token deployed on Stacks blockchain since you will need to provide the token contract.
```

Comment:
Ensure your token is deployed on the Stacks blockchain, as you will need to provide the token contract.